src/main/frontend/components/settings.cljs

```diff
@@ -694,12 +691,7 @@
     [:span.text-sm.opacity-50.my-4
      (t :settings-page/git-desc-1)]
     [:br][:br]
-    [:span.text-sm.opacity-50.my-4
```

Comment:
We should be passing in links as an argument but unfortunately we can't. Please revert these changes for reasons mentioned in `:on-boarding/android-intro-dropsync` and add the TODO